Cargo.toml

```diff
@@ -155,7 +155,7 @@ regex = { version = "1", default-features = false, features = ["std", "perf", "u
 paste = { version = "1", default-features = false }
 
 [build-dependencies]
-lalrpop = { version = "0.20.1", default-features = false }
+lalrpop = { version = "0.20", default-features = false }
```

Comment:
The lock file still seems to be using `0.20.1`. Did you maybe need to run `cargo update -p lalrpop`?

Reply:
Yes, updated in https://github.com/vectordotdev/vrl/pull/539/commits/e8fc747a9d73c7b181ea1f066b4c8cb37f730cc2